backend/api/models/ohshown_event.py

```diff
@@ -126,6 +126,7 @@ class OhshownEvent(SoftDeleteMixin):
         null=True,
     )  # 審查人
 
+    sight_see_date_time = models.DateTimeField(auto_now_add=True)
```

Comment:
does this means sight_see_date_time would be automatically recorded as current time?

I guess this is the root cause of the "sight-time same as created time" issue

Reply:
Interesting, would it be overwrote when creating the OhShownEvent object with `sight_see_date_time` parameter?

Reply:
Ah, good catch. I haven't tried but I think you are right. Thanks @yanghaochang104 for the code review!

Reply:
Ok I tried and I think 8261ec36aaa6e67928211137fb0785391e00ac35 will fix this issue. Thanks, @yanghaochang104 !